src/translations/en.json

```diff
@@ -2021,7 +2021,9 @@
     "sidebar": {
       "external_app_configuration": "App settings",
       "sidebar_toggle": "Sidebar toggle",
-      "edit_sidebar": "Edit sidebar"
+      "edit_sidebar": "Edit sidebar",
+      "edit_subtitle": "Synced on all devices",
```

Comment:
Not sure this is needed, as it also is not correct if the data is not in user data yet?

Reply:
As long as I don't hit save it's not user data but this dialog does sync it when hit save. But we can also just remove it, I am open to it. I just thought this is a nice way to inform the user it will change on all devices. Because the conformiation dialog will come just once.